hushline/routes.py

```diff
@@ -288,25 +288,30 @@ def redirect_submit_message(username: str) -> Response:
     @app.route("/delete_message/<int:message_id>", methods=["POST"])
     @authentication_required
     def delete_message(message_id: int) -> Response:
+        # Check if the user is logged in
         if "user_id" not in session:
             flash("🔑 Please log in to continue.")
             return redirect(url_for("login"))
 
+        # Fetch the user based on session user_id
         user = db.session.get(User, session.get("user_id"))
         if not user:
             flash("🫥 User not found. Please log in again.")
             return redirect(url_for("login"))
 
+        # Dynamically fetch all username IDs associated with the user
```

Comment:
This can be done in a single query if we undo my typo:

```diff
diff --git a/hushline/routes.py b/hushline/routes.py
index a111245..aae59f7 100644
--- a/hushline/routes.py
+++ b/hushline/routes.py
@@ -301,7 +301,7 @@ def init_app(app: Flask) -> None:
             db.delete(Message).where(
                 Message.id == message_id,
                 Message.username_id.in_(
-                    db.select(Username.user_id)
+                    db.select(Username.id)
                     .select_from(Username)
                     .filter(Username.user_id == user.id)
                 ),
```

Reply:
ahhhhhhhh.

Reply:
reverted, updated typo